src/store/feeValidator.ts

```diff
@@ -0,0 +1,21 @@
+import { FeeValidity } from '@src/view_models/MembershipFee';
+
+const minCentAmount = 100;
+const maxCentAmount = 100_000_00;
+
+export function validateFee( amount: number ): FeeValidity {
+
+	if ( amount < minCentAmount ) {
+		return FeeValidity.FEE_TOO_LOW;
+	}
+
+	if ( amount >= maxCentAmount ) {
+		return FeeValidity.FEE_TOO_HIGH;
+	}
+
+	return FeeValidity.FEE_VALID;
+}
+
+export function isValidFee( amount: number ): boolean {
+	return validateFee( amount ) === FeeValidity.FEE_VALID;
+}
```

Comment:
This needs a unit test that covers all the branches and tests around the edges (99, 100, 101, etc)